meta-balena-thud/recipes-devtools/cargo/cargo_0.25.0.bbappend

```diff
@@ -1,10 +0,0 @@
-FILESEXTRAPATHS_prepend := "${THISDIR}/files:"
```

Comment:
Can you please mention in the commit that we have updated meta-rust `on all devices`.
And the rust version. I was a bit confused while reviewing.

Reply:
I will.